src/Support/Helper.php

```diff
@@ -0,0 +1,37 @@
+<?php
+
+namespace Nuwave\Lighthouse\Support;
+
+use Closure;
+use Nuwave\Lighthouse\GraphQL;
+use Nuwave\Lighthouse\Exceptions\DefinitionException;
+
+class Helper
```

Comment:
I am happy to rename this class. `Util` maybe?
Or also happy to create 2 invokable classes (might be the way to go...)
I don't have any strong opinion on this @spawnia

Reply:
Not a huge fan of invokable classes. I would name it `Helpers`

Reply:
I think we should remove this class all together. 
It feels like a code smell